ibis/backends/snowflake/__init__.py

```diff
@@ -75,6 +78,7 @@ class SnowflakeExprTranslator(AlchemyExprTranslator):
 
 
 class SnowflakeCompiler(AlchemyCompiler):
+    cheap_in_memory_tables = _NATIVE_ARROW
```

Comment:
This field will always be set to True, because the value of variable `_NATIVE_ARROW` is changed only after the first query is executed.
This should probably be written as below.
```
@property
@staticmethod
def cheap_in_memory_tables():
    return _NATIVE_ARROW
```

Reply:
`_NATIVE_ARROW` is defined at import time, not after the first query is run.